gr-vocoder/lib/freedv_tx_ss_impl.cc

```diff
@@ -124,5 +106,16 @@ int freedv_tx_ss_impl::work(int noutput_items,
     return noutput_items;
 }
 
+char freedv_tx_ss_impl::get_next_tx_char(void* callback_state)
+{
+    freedv_tx_ss_impl* instance = (freedv_tx_ss_impl*)callback_state;
```

Comment:
Prefer C++ casts. [ES.49](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#es49-if-you-must-use-a-cast-use-a-named-cast)